### PR TITLE
[1LP][RFR]fix logging test 

### PR DESCRIPTION
**Pull request by shalomnaim1**
{{pytest: cfme/tests/containers/test_logging.py -v --use-provider cm-env2}}

I had some issues with this test so i was asked to fix some code.

The fixes i made are:

1. The polarion id of the test was inexact
2. I was asked to add validation to the popup window which come up after the external logging button is pushed, so i had to get the correct URL from OSE and assert it with the actual URL of the popup window.

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit b63641602e13392b68f7b41be82158cbcab212f1:

`cfme/tests/containers/test_logging.py`:
- :bangbang: Line 44:11: **E271** *multiple spaces after keyword*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/containers/provider/__init__.py [hunk 1]
```diff
@@ -295,6 +295,27 @@ class ProviderDetailsView(BaseLoggedInPage):
...
+        old_window = set(all_windows) - set([new_window])
+        browser_instance.switch_to_window(new_window)
+
+        logging_url = browser_instance.current_url
+        old_window = old_window.pop()
+
+        browser_instance.switch_to_window(old_window)
+
+        return logging_url
+
```
Comment (psav) [inline cfme/containers/provider/__init__.py]: The following I think is a better way of getting the various handles
```python
    def get_logging_url(self):
        browser_instance = browser()

        all_windows_before = browser_instance.window_handles
        appliance_window = browser_instance.current_window_handle

        self.monitor.item_select('External Logging')

        all_windows_after = browser_instance.window_handles

        new_windows = set(all_windows_before) - set(all_windows_after)

        if not new_windows:
            raise RuntimeError("No logging window was open!")

        logging_window = new_windows.pop()
        browser_instance.switch_to_window(logging_window)

        logging_url = browser_instance.current_url

        browser_instance.switch_to_window(appliance_window)

        return logging_url
```

Comment (shalomnaim1) [inline cfme/containers/provider/__init__.py]: @psav this solution it very similar to the original code, i will try use it

**Comment by shalomnaim1** [comment]
@psav  OK, that done.

**Pull request merged**